lib/authc/StormpathAssertionAuthenticator.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+
+var njwt = require('njwt');
+var AssertionAuthenticationResult = require('./AssertionAuthenticationResult');
+
+function StormpathAssertionAuthenticator(dataStore) {
```

Comment:
I fixed this for you in a new commit (bf379c508ea6e4b8d2fed8c91aee38afccc0c457).